website/project/model.py

```diff
@@ -510,13 +509,15 @@ def validate_category(value):
 
 
 def validate_title(value):
-    """Validator for Node#title. Makes sure that the value exists.
+    """Validator for Node#title. Makes sure that the value exists and is not
+    above 200 characters.
     """
     if value is None or not value.strip():
         raise ValidationValueError('Title cannot be blank.')
+    elif len(value) > 200:
```

Comment:
`if` and `elif` are identical here, since the prior block raises an exception - but for clarity, it's generally best to use `if` and space them out like so:

```python

if value is None or not value.strip():
    raise ValidationValueError('Title cannot be blank.')

if len(value) > 200:
    raise ValidationValueError('Title cannot exceed 200 characters.')

return True
```

No need to change this, I'm just being me.